storm-client/src/jvm/org/apache/storm/messaging/netty/Server.java

```diff
@@ -260,7 +260,15 @@ public void channelActive(Channel c) {
 
     @Override
     public void received(Object message, String remote, Channel channel) throws InterruptedException {
-        List<TaskMessage> msgs = (List<TaskMessage>) message;
+        List<TaskMessage> msgs;
+
+        try {
+            msgs = (List<TaskMessage>) message;
+        } catch (ClassCastException e) {
+            LOG.error("Worker netty server receive message other than the expected class List<TaskMessage>", e);
```

Comment:
Can we add remote?  I assume that might add useful debug info.

Reply:
Will this log display the original exception message string that shows the Class of the offending message?

Reply:
@agresch Remote added.
@bipinprasad Yes the exception message can be displayed just as the `ClassCastException` in the description.